bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/BookieShell.java

```diff
@@ -154,6 +155,8 @@
 /**
  * Bookie Shell is to provide utilities for users to administer a bookkeeper cluster.
  */
+@SuppressFBWarnings(value = "RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE",
```

Comment:
I see these everywhere. I don't think it is good to disable just because it doesn't work with jdk11. 

I think we should just disable spotbugs for jdk11

Reply:
jdk11  Is the current major java version, and it is LTS.
We should run spotbugs.
Javac is different, even while compiling for jdk8 release. I am spending a lot of time on this kind of issues in many projects.
If you prefer I can disable this rules on jdk11 but not completely spotbugs

Reply:
please disable these rules on jdk11 only.

Reply:
Okay, it will be tricky but it is worth.

Reply:
@sijie 
I am re-thinking about disabling this rule only on JDK11+....
I feel it is not worth such complication. We should have two different findbugs configuration files, and with JDK11+ the problem will stay forever.

In Apache Kafka they have disabled it at all
https://github.com/apache/kafka/pull/5943/files

Reply:
@sijie 
The same we did in Pravega and in ZooKeeper
https://github.com/pravega/pravega/commit/fbaa1577385bab730e6c2f10d9fcda48478a2162

https://github.com/apache/zookeeper/commit/b752ef66876a141035a42f30aad69e3166cad746#diff-b4a4d2c7fb2ac879fb126283ee65fd07

I am asking on JDK compiler dev mailing list for solutions, but any solution won't be feasible until jdk13, if any solution will be ever adopted

Reply:
@eolivelli I would still prefer not disabling those rules for non-jdk11 environments. we can disable spotbugs for jdk11 only.

Reply:
@sijie okay, I will update the patch.

Let's disable spotbugs from JDK11+ at all